src/openrct2/actions/LargeSceneryPlaceAction.hpp

```diff
@@ -364,21 +365,21 @@ DEFINE_GAME_ACTION(LargeSceneryPlaceAction, GAME_COMMAND_PLACE_LARGE_SCENERY, La
             if (surfaceElement == nullptr)
                 continue;
 
-            int32_t height = surfaceElement->GetBaseZ();
+            int32_t baseZ = surfaceElement->GetBaseZ();
             int32_t slope = surfaceElement->GetSlope();
 
-            if (slope & 0xF)
+            if ((slope & TILE_ELEMENT_SLOPE_ALL_CORNERS_UP) != TILE_ELEMENT_SLOPE_FLAT)
             {
-                height += 16;
-                if (slope & 0x10)
+                baseZ += 16;
```

Comment:
LAND_HEIGHT_STEP

Reply:
One day I'll go round and refactor all the million instances of this code to call a function.